### DISCUSSION TITLE
Can you detect the serverless stage in php?

### DISCUSSION THREAD

**Discussion by ctessmer** [brefphp/bref#1832] [Support]
I'd like to be able to detect stage (`prod` || `dev`) in my php code. 

1. Is this possible? 
2. If so how? 
3.  Bonus: Are any of the environment variables set in `serverless.yaml` available to read in php code?

## Example

For instance, if I want the css served in my production env to be minified:

```php
<link rel="stylesheet" type="text/css" href="/<?= (isProduction()) ? 'css/production.min.css' : 'css/development.css') ?>" />
```

Where `isProduction()` is a function that "detects the stage"  (`prod` || `dev`) by reading it directly or reading the app name (`site-prod-php` || `site-dev-php`) or some other serverless value.

## TL;DR:

**What values can I read in php, if any, from the serverless environment?**

Thanks!
